documentation/src/main/asciidoc/topics/ref_transcoders.adoc

```diff
@@ -0,0 +1,20 @@
+[id='transcoders_{context}']
+= Transcoders
+Transcoders convert entries to and from different media types.
+
+The default API is `org.infinispan.commons.dataconversion.Transcoder`:
+
+[source,java]
+----
+include::code_examples/Transcoder.java[]
+----
+
+[NOTE]
+====
+For embedded caches, you must add the following to your project:
```

Comment:
I thought we weren't going to promote embedded server mode anymore? @tristantarrant @gustavonalle

Reply:
yeah, let's just drop this detail and stop making all the details available. if someone on the community side needs it, they can probably just ask how and get better guidance.